net/remote/remote-node.sh

```diff
@@ -388,7 +387,7 @@ EOF
         args+=(--keypair config/validator-identity.json)
       fi
 
-      multinode-demo/delegate-stake.sh "${args[@]}"
+      multinode-demo/delegate-stake.sh "${args[@]}" "$internalNodesStakeLamports"
```

Comment:
if internalNodesStakeLamports is the empty string, you don't want to pass it to delegate-stake (which has a default value)

Reply:
actually, we check -n, so is equivalent...

in delegate_stake.sh:
```
if [[ -n ${positional_args[0]} ]]; then
  stake_lamports=${positional_args[0]}
fi
```

Reply:
Right, so it's the empty string, we will use the default in delegate_stake.sh.  If it's not empty, we override the default.  From your two comments together, it's not clear to me if you are okay with the way this code is written or not?

Reply:
am ok